generate_dot_env_dot_example.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+with open(".env") as f:
+    lines = f.read().strip().split("\n")
+
+
+out_lines = []
+for line in lines:
+    first_part = line.split("=")[0]
+    out_lines.append(first_part + "=" + "changeme")
+
+if os.path.exists("test.env.example"):
+    raise ValueError("test.env.example already exists")
```

Comment:
what is the point of this? if you want to add a new line to .env, wouldnt you want this to delete/recreate the old test.env.example?